Jenkinsfile

```diff
@@ -1,85 +1,6 @@
 #!/usr/bin/env groovy
 
-REPOSITORY = 'email-alert-service'
-DEFAULT_SCHEMA_BRANCH = 'deployed-to-production'
-
 node {
   def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
-
-  properties([
-    buildDiscarder(
-      logRotator(
-        numToKeepStr: '10')
-      ),
-    [$class: 'RebuildSettings', autoRebuild: false, rebuildDisabled: false],
-    [$class: 'ThrottleJobProperty',
-      categories: [],
-      limitOneJobWithMatchingParams: true,
-      maxConcurrentPerNode: 1,
-      maxConcurrentTotal: 0,
-      paramsToUseForLimit: REPOSITORY,
-      throttleEnabled: true,
-      throttleOption: 'category'],
-    [$class: 'ParametersDefinitionProperty',
-      parameterDefinitions: [
-        [$class: 'BooleanParameterDefinition',
-          name: 'IS_SCHEMA_TEST',
-          defaultValue: false,
-          description: 'Identifies whether this build is being triggered to test a change to the content schemas'],
-        [$class: 'StringParameterDefinition',
-          name: 'SCHEMA_BRANCH',
-          defaultValue: DEFAULT_SCHEMA_BRANCH,
-          description: 'The branch of govuk-content-schemas to test against']]
-    ],
-  ])
-
-  try {
-    govuk.initializeParameters([
-      'IS_SCHEMA_TEST': 'false',
-      'SCHEMA_BRANCH': DEFAULT_SCHEMA_BRANCH,
-    ])
-
-    if (!govuk.isAllowedBranchBuild(env.BRANCH_NAME)) {
-      return
-    }
-    
-    stage("Checkout") {
-      checkout scm
-    }
-
-    stage("git merge") {
-      govuk.mergeMasterBranch()
-    }
-
-    stage("bundle install") {
-      govuk.bundleApp()
-    }
-
-    stage("Delete queue") {
-      sh("GOVUK_ENV=test bundle exec bin/delete_queue")
```

Comment:
Does this still need to run? If it's part of the test setup, could it be added to the rake default task?